.circleci/config.yml

```diff
@@ -598,11 +598,15 @@ jobs:
       - run:
           name: Lint DockerFile changes
           command: |
-            #eval `.circleci/get_pr_info.sh -b`
-            #if  [[ -n "$BASE_GITHASH" ]] && [[ -n "$CHANGED_FILE_OUTPUTFILE" ]] && [[ "wc -l < $CHANGED_FILE_OUTPUTFILE" != 0 ]]; then
-            #  CHANGED_DOCKER_FILES=$(cat "$CHANGED_FILE_OUTPUTFILE" | grep Dockerfile | tr '\n' ' ')
-            #  hadolint -c .lintrules/hadolint.yaml $CHANGED_DOCKER_FILES || true
-            #fi
+            target_info=$(.circleci/get_pr_info.sh -b)
+            echo "$target_info"
+            eval "$target_info"
```

Comment:
Maybe try `set -ex` with the original `eval $(blah)` so that we can what the env vars it tired to set and avoid this ad-hoc?